pkg/cache/clusterqueue.go

```diff
@@ -268,7 +267,9 @@ func (c *clusterQueue) inactiveReason() (string, string) {
 		// This doesn't need to be gated behind, because it is empty when the gate is disabled
 		if len(c.multipleSingleInstanceControllersChecks) > 0 {
 			reasons = append(reasons, kueue.ClusterQueueActiveReasonMultipleSingleInstanceControllerAdmissionChecks)
-			for _, controller := range utilmaps.SortedKeys(c.multipleSingleInstanceControllersChecks) {
+			controllerKey := slices.Collect(maps.Keys(c.multipleSingleInstanceControllersChecks))
+			slices.Sort(controllerKey)
+			for _, controller := range controllerKey {
```

Comment:
```suggestion
			for _, controller := range slices.Sorted(maps.Keys(c.multipleSingleInstanceControllersChecks)) {
```
Will it work like this?